x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1159,202 +1164,190 @@ describe('Task Runner', () => {
-        );
-        rulesClient.get.mockResolvedValue({
-          ...mockedRuleTypeSavedObject,
-          notifyWhen: 'onActionGroupChange',
-        });
-        encryptedSavedObjectsClient.getDecryptedAsInternalUser.mockResolvedValue({
-          id: '1',
-          type: 'alert',
-          attributes: {
-            apiKey: Buffer.from('123:abc').toString('base64'),
-            enabled: true,
-          },
-          references: [],
-        });
-        await taskRunner.run();
-        expect(enqueueFunction).toHaveBeenCalledTimes(1);
-      }
+        },
+        customTaskRunnerFactoryInitializerParams
+      );
+      rulesClient.get.mockResolvedValue({
+        ...mockedRuleTypeSavedObject,
+        notifyWhen: 'onActionGroupChange',
+      });
+      encryptedSavedObjectsClient.getDecryptedAsInternalUser.mockResolvedValue({
+        id: '1',
+        type: 'alert',
+        attributes: {
+          apiKey: Buffer.from('123:abc').toString('base64'),
+          enabled: true,
+        },
+        references: [],
+      });
+      await taskRunner.run();
+      expect(enqueueFunction).toHaveBeenCalledTimes(1);
+    }
   );
 
-  testAgainstEphemeralSupport(
-    'actionsPlugin.execute is called when notifyWhen=onActionGroupChange and alert state subgroup has changed',
-    (
-        customTaskRunnerFactoryInitializerParams: TaskRunnerFactoryInitializerParamsType,
-        enqueueFunction: (options: ExecuteOptions) => Promise<void | RunNowResult>
-      ) =>
-      async () => {
-        customTaskRunnerFactoryInitializerParams.actionsPlugin.isActionTypeEnabled.mockReturnValue(
-          true
-        );
-
-        customTaskRunnerFactoryInitializerParams.actionsPlugin.isActionExecutable.mockReturnValue(
-          true
-        );
-        ruleType.executor.mockImplementation(
-          async ({
-            services: executorServices,
-          }: AlertExecutorOptions<
-            AlertTypeParams,
-            AlertTypeState,
-            AlertInstanceState,
-            AlertInstanceContext,
-            string
-          >) => {
-            executorServices
-              .alertInstanceFactory('1')
-              .scheduleActionsWithSubGroup('default', 'subgroup1');
-          }
-        );
-        const taskRunner = new TaskRunner(
-          ruleType,
-          {
-            ...mockedTaskInstance,
-            state: {
-              ...mockedTaskInstance.state,
-              alertInstances: {
-                '1': {
-                  meta: {
-                    lastScheduledActions: {
-                      group: 'default',
-                      subgroup: 'newSubgroup',
-                      date: new Date().toISOString(),
-                    },
+  test.each(ephemeralTestParams)(
+    'actionsPlugin.execute is called when notifyWhen=onActionGroupChange and alert state subgroup has changed %s',
+    async (nameExtension, customTaskRunnerFactoryInitializerParams, enqueueFunction) => {
```

Comment:
Suggest adding an expect to this test to ensure the correct number of triggered actions is logged to the event log.